pandas/_libs/tslibs/offsets.pyx

```diff
@@ -2313,12 +2313,28 @@ cdef class YearEnd(YearOffset):
 
 cdef class YearBegin(YearOffset):
     """
-    DateOffset increments between calendar year begin dates.
+    DateOffset of one year at beginning.
```

Comment:
this matches the wording on `MonthBegin`

wondering if there's a clearer wording, but that can be for a separate PR